src/core/agent/services/identifierService.ts

```diff
@@ -531,6 +531,13 @@ class IdentifierService extends AgentService {
     }
     return isDone;
   }
+
+  async rotateIdentifier(metadata: IdentifierMetadataRecord) {
+    if (!metadata.signifyName) {
```

Comment:
Can we add a check before this to check if the identifier is a DID or AID and return a different error message if it's a DID? (We can't rotate any DID we support right now)